prismic_remapper/_eg-promo-split/remap.js

```diff
@@ -0,0 +1,23 @@
+// Accessing the promo of linked content is impossible at the moment as you
+// can't query `fetchLinks` on Slices. This splits the promo out into
+// `promoText` And promoImage, which you can both fetch with `fetchLinks`
+
+module.exports = {
+  filter: ({filename, doc}) => {
```

Comment:
you _could_ use the object function shorthand (`filter({filename, doc}) {}`) and save yourself all of three keystrokes(!)